bokeh/io.py

```diff
@@ -37,6 +37,8 @@
 from .util.notebook import load_notebook, publish_display_data, get_comms
 from .util.string import decode_utf8
 from .util.serialization import make_id
+from .resources import INLINE
+from .core.templates import ZEPPELIN_FILE
```

Comment:
This should be wrapped up in a function. e.g. `bokeh.util.notebook.load_zeppelin` In doing so, you can also avoid the import of `INLINE` here. The `load_zeppelin` function should be consistent with the existing `bokeh.util.notebook.load_notebook` (which should be renamed to `bokeh.util.notebook.load_jupyter`) in that passing `None` for a resources arg has a default meaning. 

I'd also carefully consider whether the default should be inline. We did that originally for jupyter notebooks and changed to can default after a heavy user request to do so. Inline resources will make all notebooks substantially larger

Also imports and import lines should be in alphabetical order throughout.